spec/controllers/annotation_categories_controller_spec.rb

```diff
@@ -475,7 +475,7 @@
 
       expect(response.status).to eq(302)
       # One annotation category was created, and one has an error.
```

Comment:
You should update this comment to make it clear that there is a valid annotation category in the file, but it isn't saved.